### DISCUSSION TITLE
Not syncing for 3 days now

### DISCUSSION THREAD

**Discussion by chiaboi** [Chia-Network/chia-blockchain#6440] [Support]
My account hasn't been able to sync in 3 days. Also 3 days ago I added a new HDD. Coincidence? I think so? Either way, I can't sync. Restarted Chia multiple times. Restarted PC multiple times. Stopped plotting. Nothing. 

Right now my status is just "Not Synced". It varies between that and "Syncing" which it never does. 

Any ideas? Help? Where to start? Thanks.

**Comment by chiaboi** [comment]
Is this the wrong forum?

**Comment by gromran** [comment]
which version?

**Reply by chiaboi** [reply]
1.1.4

**Reply by antonello22** [reply]
well you have to update to at least 1.6 ... the 1.7 for now does not work for me....

**Comment by loppefaaret** [comment]
please update!

**Reply by chiaboi** [reply]
OMG that's embarrassing. I'm getting soft with all of these automatic updates.

Thanks!

**Comment by BrandtH22** [comment]
@chiaboi , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
